src/ga_tx.cpp

```diff
@@ -555,14 +555,20 @@ namespace sdk {
 
                 // Add all outputs and compute the total amount of satoshi to be sent
                 amount required_total{ 0 };
+                uint32_t greedy_index = NO_CHANGE_INDEX;
 
                 if (num_addressees) {
+                    int addressee_index = 0;
                     for (auto& addressee : *addressees_p) {
                         const auto addressee_asset_id = asset_id_from_json(net_params, addressee);
                         if (addressee_asset_id == asset_id) {
                             required_total += add_tx_addressee(session, net_params, result, tx, addressee);
                             reordered_addressees.push_back(addressee);
+                            if (addressee.value("greedy", false)) {
```

Comment:
booleans in json should be named `is_xxx`, `has_xxx` etc.

There is no check here for multiple greedy outputs, should that error?

Reply:
Changed to `is_greedy`